Source/Examples/Speciality IO Example/Speciality IO Example.md

```diff
@@ -0,0 +1,31 @@
+# **The Speciality IO Example contains 2 sections:**
+
+## 1) The project used to generate a bitfile for the FPGA Addon Custom device
+   - Namely, the Speciality IO FPGA.lvproj ;
+   - The bitfile is generated for a cRIO-9039 target;
+   - Users can add a different target in the project and use the Speciality IO Example.vi to generate a bitfile for another target;
```

Comment:
Please add another bullet here: For more information about FPGA Speciality IO, visit the official repo: https://github.com/ni/niveristand-fpga-addon-speciality-io